src/EnergyPlus/HVACDXSystem.cc

```diff
@@ -2149,8 +2168,7 @@ namespace HVACDXSystem {
                             PartLoadFrac = 0.0;
                         }
                     } else if (SELECT_CASE_var == Coil_CoolingAirToAirVariableSpeed) { // Coil:Cooling:DX:VariableSpeed
-                        // variable-speed air-to-air cooling coil, begin -------------------------
-                        // Get no load result
+
```

Comment:
These changes are hard to see here but the jist is that we used to test no load and full load before any checks on capacity vs load. Now only check no load result before testing full load result. No need to test full load result if no load output meets set point.

Reply:
You're right, the changes are hard to see.  I mean the story you've narrated here looks like the standard approach in HVAC equipment.  Is there any significant change to this stuff related to RH control?  Or is it only the changes in the residual function that fix RH control?